art/step/steps.py

```diff
@@ -17,8 +19,8 @@ class EvaluateBaseline(Step):
     name = "Evaluate Baseline"
     description = "Evaluates a baseline on the dataset"
 
-    def __init__(self, baseline: ArtModule, datamodule: LightningDataModule):
-        trainer = Trainer(accelerator=baseline.device.type)
+    def __init__(self, baseline: ArtModule, datamodule: LightningDataModule, logger: Optional[Union[Logger, Iterable[Logger], bool]] = None):
```

Comment:
I don't think logging anything here makes any sense. Logger should be used only in the steps when logging to is appropriate e.i. longer training.

Reply:
Right now I would agree, but if we would want to use logger to communicate with dashboard. (Add dashboard as a logger, to show updates more live) than it would make sense, but right now not.

Reply:
direct communication beetween dashboard app and steps + live updates seems like very hard thing to do. Isn't communication through files enough? And providing refresh button in the dashboard?

Reply:
For now it is enough, but maybe in v5...?